pkg/apis/v1/nodeclaim_defaults.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import "context"
+
+// SetDefaults for the NodeClaim
+func (in *NodeClaim) SetDefaults(_ context.Context) {}
```

Comment:
Needed now? I think we're planning to drop the webhooks so we should be able to remove these files entirely